src/minimega/vm.go

```diff
@@ -620,8 +635,10 @@ func (vm *BaseVM) Info(field string) (string, error) {
 // setState updates the vm state, and write the state to file. Assumes that the
 // caller has locked the vm.
 func (vm *BaseVM) setState(s VMState) {
-	log.Debug("updating vm %v state: %v -> %v", vm.ID, vm.State, s)
+	now := time.Now()
+	log.Debug("updating vm %v state: %v -> %v, at time %s", vm.ID, vm.State, s, now)
```

Comment:
`log` includes the current time -- it doesn't seem necessary to add it to the message.